static/admin/config.yml

```diff
@@ -0,0 +1,356 @@
+backend:
+  name: git-gateway
+  repo: ec-europa/ecl-twig
+  branch: develop
+  squash_merges: true
+  commit_messages:
+    create: 'feat({{collection}}): create {{collection}} “{{slug}}”'
+    update: 'feat({{collection}}): update {{collection}} “{{slug}}”'
+    delete: 'feat({{collection}}): delete {{collection}} “{{slug}}”'
+    uploadMedia: 'Upload “{{path}}”'
+    deleteMedia: 'Delete “{{path}}”'
+
+media_folder: static/images
+publish_mode: editorial_workflow
```

Comment:
Unnecessary I think. There won't be too many people doing too many things I think.

Reply:
Sure, the whole cms thingy is unnecessary here on twig, i'd say. The idea is to use this configuration to test in advance what would be the overall experience of an "editor" of these documents in ECL, including the github workflow, that's why i set it here.